tests/instrumentation-framework/pom.xml

```diff
@@ -38,15 +38,26 @@
     </dependency>
 
     <dependency>
-      <groupId>com.spotify</groupId>
-      <artifactId>docker-client</artifactId>
-      <version>${version.com.spotify.docker-client}</version>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>${version.com.github.docker-java.docker-java}</version>
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
       <version>${version.org.slf4j}</version>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.5</version>
```

Comment:
Replace it with ${version.commons.io}. It might even already exist at one of the parents.